--- config/cmake/Modules/Autotools/ArchiveX/cx_swig_python_module.cmake	2018-11-07 17:46:02.000000000 +0000
+++ config/cmake/Modules/Autotools/ArchiveX/cx_swig_python_module.cmake	2018-12-12 15:43:00.000000000 +0000
@@ -81,13 +81,16 @@
     PROPERTIES
       OUTPUT_NAME _${ARG_MODULE})
   if ( APPLE )
+    swig_link_libraries( ${target_name} ${ARG_LINK_LIBRARIES} )
     set_target_properties(${SWIG_MODULE_${target_name}_REAL_NAME}
       PROPERTIES
         INSTALL_RPATH "${CMAKE_INSTALL_FULL_LIBDIR}"
-        BUILD_WITH_INSTALL_RPATH TRUE )
+        BUILD_WITH_INSTALL_RPATH TRUE
+        LINK_FLAGS "-undefined dynamic_lookup"
+    )
+  else ( )
+    swig_link_libraries( ${target_name} ${${ARG_PREFIX}_LIBRARIES} ${ARG_LINK_LIBRARIES} )
   endif( )
-  # cm_msg_debug_variable(${ARG_PREFIX}_LIBRARIES)
-  swig_link_libraries( ${target_name} ${${ARG_PREFIX}_LIBRARIES} ${ARG_LINK_LIBRARIES} )

   #----------------------------------------------------------------------
   # Compile .py files
